rpc_util.go

```diff
@@ -189,6 +189,13 @@ type EmptyCallOption struct{}
 func (EmptyCallOption) before(*callInfo) error      { return nil }
 func (EmptyCallOption) after(*callInfo, *csAttempt) {}
 
+// RegisteredMethodCallOption is a call option that specifies that a call comes
+// from a registered method.
+type RegisteredMethodCallOption struct{}
```

Comment:
Change to function that returns a call option

Reply:
More accurately: add a function that returns this (as a `CallOption`) and keep this.  Please try to docstring what it does (nothing, except is a signal to other code we integrate)

Reply:
Done.